apps/web/src/utils/page-visibility.ts

```diff
@@ -17,36 +17,38 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-var hidden, visibilityChange;
-if (typeof document.hidden !== "undefined") {
+let hidden: string, visibilityChange: string;
```

Comment:
Use strict typing here. Actually turn this into a function that returns `hidden` & `visibilityChange` with strict types.